config/rbac/submariner-diagnose/role.yaml

```diff
@@ -10,6 +10,7 @@ rules:
     resources:
       - pods
     verbs:
+      - delete
```

Comment:
Can you be more specific about what needs this permission?

Reply:
This function deletes the pods.
https://github.com/submariner-io/subctl/blob/aedd7a6e8e43e95b809f0caa96325f45213b46e9/internal/pods/schedule.go#L182
And it's being called by tests. After pods spawned by the test are done, they are deleted by the test the spawned them